RedHatInsights/insights-remediations-frontend
# feat(ESSNTL-4496): Replace SmartManager w/ satellite

# Description
Associated Jira ticket: # ([ESSNTL-4496)](https://issues.redhat.com/browse/ESSNTL-4496)
This replaces all references of Red Hat Smart Manager with Satellite. I've left permissions as is for the time being.

# How to test the PR
Just skim the code and UI for any references to smart manager and verify there are none the user can see.

# Checklist:

- [ x] PR has a short description
- [x ] Tests for the changes have been added
